code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -305,25 +363,46 @@ Bonus
 //////////////////////////////////////
 */
 
-/datum/symptom/heal/dna
-
-	name = "Deoxyribonucleic Acid Restoration"
-	desc = "The virus repairs the host's genome, purging negative mutations."
+/datum/symptom/heal/radiation
+	name = "Radioactive Resonance"
+	desc = "The virus uses radiation to fix damage through dna mutations."
 	stealth = -1
-	resistance = -1
+	resistance = -2
 	stage_speed = 0
 	transmittable = -3
-	level = 5
-	symptom_delay_min = 3
-	symptom_delay_max = 8
-	threshold_desc = "<b>Stage Speed 6:</b> Additionally heals brain damage.<br>\
-					  <b>Stage Speed 11:</b> Increases brain damage healing."
-
-/datum/symptom/heal/dna/Heal(mob/living/carbon/M, datum/disease/advance/A)
-	var/amt_healed = 2 * (power - 1)
-	M.adjustBrainLoss(-amt_healed)
-	//Non-power mutations, excluding race, so the virus does not force monkey -> human transformations.
-	var/list/unclean_mutations = (GLOB.not_good_mutations|GLOB.bad_mutations) - GLOB.mutations_list[RACEMUT]
-	M.dna.remove_mutation_group(unclean_mutations)
-	M.radiation = max(M.radiation - (2 * amt_healed), 0)
+	level = 6
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	threshold_desc = "<b>Stage Speed 6:</b> Increases healing."
+
+/datum/symptom/heal/radiation/CanHeal(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	switch(M.radiation)
+		if(0)
+			return FALSE
+		if(1 to 350)
+			return 0.25

```

Comment:
It'd probably be best if you can't heal unless you're getting some negative from the radiation.
`if(0 to RAD_MOB_SAFE) return FALSE`